### DISCUSSION TITLE
mime type of uploading file is application/octet-stream fails.

### DISCUSSION THREAD

**Discussion by avinashtelsa** [Laravel-Backpack/community-forum#950] [General] [answered]
In my code, files with the MIME type "application/pdf" are successfully uploaded, but PDF files with the MIME type "application/octet-stream" are not getting uploaded. I need to fix that.
![image](https://github.com/Laravel-Backpack/community-forum/assets/166697230/84ae7bd1-715d-4aa1-95c6-84aca6fc9513)
![image](https://github.com/Laravel-Backpack/community-forum/assets/166697230/f2240144-49a9-4842-b914-1d00b3e57ca0)

**Comment by karandatwani92** [comment, marked as answer]
Hey @avinashtelsa 

This question is related to Laravel Validations, not Backpack. I think `'mimes:pdf,application/octet-stream'` may work here.

**Reply by avinashtelsa** [reply]
Hello,

I tried this but it didn't work.

**Reply by karandatwani92** [reply]
I googled a bit, and it seems related to server configuration.
https://stackoverflow.com/a/77493150/4869406
https://developer.mozilla.org/en-US/docs/Learn/Server-side/Configuring_server_MIME_types

Sorry, I can't help more with this.
